app/models/manageiq/providers/google/cloud_manager/event_catcher/runner.rb

```diff
@@ -0,0 +1,74 @@
+class ManageIQ::Providers::Google::CloudManager::EventCatcher::Runner <
+  ManageIQ::Providers::BaseManager::EventCatcher::Runner
+
+  # Request the event monitor to stop running.
+  def stop_event_monitor
+    stop_running
+  end
+
+  # Start monitoring for events. This method blocks forever until
+  # #stop_event_monitor is called.
+  def monitor_events
+    _log.info "#{log_prefix} Monitoring for events"
+
+    start_running
+    event_monitor_running
+    while running?
+      heartbeat
+      events = event_producer.poll
+      heartbeat
+
+      _log.info "#{log_prefix} Received #{events.size} events" unless events.empty?
+      events.each do |event|
+        _log.debug "#{log_prefix} Event: #{event}"
+        @queue.enq event if event_requires_refresh?(event)
+      end
+      sleep(20)
+    end
+    _log.info "#{log_prefix} Exiting event monitor loop"
+  ensure
+    stop_running
+  end
+
+  private
+
+  def process_event(event)
```

Comment:
you should add a `filtered` here, like [here](https://github.com/ManageIQ/manageiq/blob/772fa7f7e68f891ed660d4b6cbda8ac91f7e668a/gems/manageiq-providers-amazon/app/models/manageiq/providers/amazon/cloud_manager/event_catcher/runner.rb#L30-L32)

the blacklisting is setup [here](https://github.com/ManageIQ/manageiq/blob/be619aa352d9d6688c6f1c68b9e93e6cd0c49e95/gems/manageiq-providers-amazon/app/models/manageiq/providers/amazon/network_manager.rb#L67-L73)